src/main/java/de/unistuttgart/iste/gits/reward/controller/SubscriptionController.java

```diff
@@ -20,9 +21,9 @@ public class SubscriptionController {
     private final RewardService rewardService;
 
     /**
-     * Event handler for the content-progressed event
+     * Event handler for the user-progress-updated event
      */
-    @Topic(name = "content-progressed", pubsubName = "gits")
+    @Topic(name = "user-progress-updated", pubsubName = "gits")
```

Comment:
this new event is thrown by the content service after it processed the "content-progressed" event. Data is the same